blog/.vuepress/theme/layouts/Post.vue

```diff
@@ -77,6 +88,16 @@ export default {
       return {
         "background-image": `url(${this.$withBase(this.current.image)})`
       };
+    },
+
+    gravatar () {
```

Comment:
I would suggest to move this logic to `https://github.com/alexander-heimbuch/vuepress-theme-casper/blob/master/blog/.vuepress/theme/store/utils.js#L4`